xclim/indicators/atmos/_temperature.py

```diff
@@ -1445,3 +1461,38 @@ def cfcheck(self, tas, snd=None):
     compute=indices.hardiness_zones,
     parameters={"method": "usda"},
 )
+
+chill_portions = TempHourly(
+    title="Chill portions",
+    identifier="cp",
+    units="",
+    cell_methods="time: sum",
+    description="Chill portions are a measure to estimate the bud breaking potential of different crops. "
+    "The constants and functions are taken from Luedeling et al. (2009) which formalises "
+    "the method described in Fishman et al. (1987). ",
+    abstract="Chill portions are a measure to estimate the bud breaking potential of different crops. "
+    "The constants and functions are taken from Luedeling et al. (2009) which formalises "
+    "the method described in Fishman et al. (1987). "
+    "The model computes the accumulation of cold temperatures in a two-step process. "
+    "First, cold temperatures contribute to an intermediate product that is transformed to a chill portion "
+    "once it exceeds a certain concentration. The intermediate product can be broken down at higher temperatures "
+    "but the final product is stable even at higher temperature. "
+    "Thus the dynamic model is more accurate than other chill models like the Chilling hours or Utah model, "
+    "especially in moderate climates like Israel, California or Spain.",
+    long_name="Chill portions after the Dynamic Model",
+    allowed_periods=["A"],
+    compute=indices.chill_portions,
+)
+
+chill_units = TempHourlyWithIndexing(
+    title="Chill units",
+    identifier="cu",
+    units="",
+    cell_methods="time: sum",
+    description="Chill units are a measure to estimate the bud breaking potential of different crops based on the Utah model developed in "
+    "Richardson et al. (1974). The Utah model assigns a weight to each hour depending on the temperature recognising that high temperatures can "
+    "actually decrease the potential for bud breaking.",
+    long_name="Chill units after the Utah Model",
+    allowed_periods=["Y"],
```

Comment:
I am a little confused about the usage of `allowed_periods=['A']` I got an error and looking at the source code I think it should be `allowed_periods=["Y"]`?

Reply:
Indeed, it should be "Y". The "A" in this files are leftovers bugs from when pandas changed its syntax. Woops.